### ISSUE TITLE
Background video embed component

### ISSUE THREAD

**Issue by danielfmiranda** [mozilla/foundation.mozilla.org#6981]
Similar to the full-width video component we already have from #6834, which allows us to embed a youtube video at regular, wide, and full-widths on the wagtail blog page, we want a new component that works with either the vimeo API or self hosting the media through S3,vand allows us to embed a looping, muted, video following the design made in #6834 or #6675. 

This would include:
- Investigating/creating a proof of concept using vimeo. If not possible, investigate self-hosting the media
- Creating the new component on the wagtail/backend side
- Creating the new front end template, and adding required js to get the video player to work. 

Note: Once the header-hero video ticket #6675  is merged into the codebase, you can probably copy/use the template for reference to follow the requested design.
